sdk/core/core-amqp/samples/cbsAuth.ts

```diff
@@ -52,11 +53,15 @@ export const connectionContext = ConnectionContextBase.create(parameters);
  */
 export async function authenticate(
   audience: string,
-  closeConnection = false
+  closeConnection: boolean = false
```

Comment:
This function need not be exported. It is meant to be used just in this sample right?
Also, `closeConnection: boolean = false` can be `closeConnection?: boolean`